src/azurelinux/manifest.json

```diff
@@ -943,8 +941,21 @@
               "variant": "v8"
             }
           ]
+        },
+        {
+          "platforms": [
+            {
+              "architecture": "amd64",
+              "dockerfile": "src/azurelinux/3.0/build-deps",
+              "os": "linux",
+              "osVersion": "azurelinux3.0",
+              "tags": {
+                "azurelinux-3.0-build-deps-amd64": {}
```

Comment:
I'm not sure the `deps` suffix really adds anything. Every image in this repo is kind of a `deps` image already.

Reply:
That's true. I was motivate to ensure people realized there was no SDK inside. However, I have low care factor on that.

Reply:
I changed it.